src/components/views/auth/InteractiveAuthEntryComponents.tsx

```diff
@@ -440,11 +481,54 @@ export class EmailIdentityAuthEntry extends React.Component<IEmailIdentityAuthEn
         } else {
             return (
                 <div className="mx_InteractiveAuthEntryComponents_emailWrapper">
-                    <p>{ _t("A confirmation email has been sent to %(emailAddress)s",
+                    <AuthHeader
+                        title={_t("Check your email to continue")}
+                        icon={<img
+                            src={EmailPromptIcon}
+                            alt={_t("Unread email icon")}
+                            width={16}
+                        />}
+                        hideServerPicker={true}
```

Comment:
Now here we both say hideServerPicker and ignore the `serverPicker` prop - feels like two conflated mechanisms for one ability